### PR TITLE
Bug 1806915: remove runlevel=1 from service-ca-operator owned namespaces

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

**Comment by stlaz** [comment]
/retry
interesting, SCC admission fail, aside from other operators failing as well

**Comment by stlaz** [comment]
/hold
testing with the operand's SA being able to access all SCCs

File: bindata/v4.0.0/controller/role.yaml [hunk 1]
```diff
@@ -4,6 +4,14 @@ metadata:
...
   namespace: openshift-service-ca
 rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - restricted
+  verbs:
+  - use
```
Comment (sttts) [inline bindata/v4.0.0/controller/role.yaml]: "use" is a verb?

Comment (stlaz) [reply bindata/v4.0.0/controller/role.yaml]: yes, it's used in the SCC admission: https://github.com/openshift/origin/blob/3c26fadaf29adfeefab38907792ebb6167526d3c/vendor/github.com/openshift/apiserver-library-go/pkg/securitycontextconstraints/sccmatching/matcher.go#L74-L74

**Comment by stlaz** [comment]
/retest

The dependency loop eventually got down to cluster-policy-controller not being able to connect to KAS.

https://github.com/openshift/cluster-etcd-operator/pull/233 removes the dependency on DNS, let's see if it can be as good as solving our little dependency loop problem.

**Comment by stlaz** [comment]
this won't solve the problem with upgrades though, that needs library-go fixing, because `ApplyNamespace` only merges NS meta instad of actually applying it.

**Comment by stlaz** [comment]
/remove-lifecycle stale
in theory, this could now work

**Comment by stlaz** [comment]
/retest
infra? AWS internal errors?

**Comment by stlaz** [comment]
/test e2e-aws-operator
yay, the control plane came up

**Pull request merged**